### ISSUE TITLE
Change the color of the Card Catalog on schema page

### ISSUE THREAD

**Issue by dnaidionov** [cardstack/cardstack#1357]
Color should be as close to #413E4E as accessibility features allow

**Comment by burieberry** [comment]
I realized this is being address in #1335 but I don't mind if one this is merged separately. The only change is in `catalog.css` file's `.ch-catalog` class:

```
background-color: var(--ch-dark-background);
color: var(--ch-light);
```
